### DISCUSSION TITLE
Notification "HOMEMATICIP_LOCAL-Ping/Pong Mismatch on Interface"

### DISCUSSION THREAD

**Discussion by WegnerDan** [SukramJ/aiohomematic#1108] [Q&A] [answered]
![image](https://github.com/danielperna84/hahomematic/assets/6908247/4ff3d173-8b27-467f-89cb-793ff3674690)

> There is a mismatch between send ping events and received pong events for HA instance homematic. Looks like you are running multiple instances of HA with the same instance name configured for this integration. Re-add one instance! Otherwise one HA instance will not receive update events from your CCU.

I got this notification today. I am not using multiple HA instances and I don't have any problems with homematic devices in home assistant, so I think this is just a false positive on that notification. Is there anything else, that could cause this?

I found these pull requests: https://github.com/danielperna84/hahomematic/pull/1099 https://github.com/danielperna84/hahomematic/pull/1103

**Comment by SukramJ** [comment, marked as answer]
Restart your ccu? Maybe somethong is stuck there.

**Reply by WegnerDan** [reply]
After restarting CCU last week, I got no other notifications, so I guess it's working correctly.
